packages/amplify-e2e-tests/schemas/many-to-many.graphql

```diff
@@ -0,0 +1,12 @@
+type Post @model @auth(rules: [{ allow: public }]) {
```

Comment:
Can we disambiguate the file names between `many-to-many` and `many_to_many`? Preference would be to just reuse them, but it looks like you've got auth rules in one and not the other, so at least a clarification on what the schemas are for would be helpful.

Reply:
Accidental addition. Only `many-to-many.graphql` is needed.